O365/sharepoint.py

```diff
@@ -236,8 +240,41 @@ def get_list_columns(self):
 
         return [self.list_column_constructor(parent=self, **{self._cloud_data_key: column})
                 for column in data.get('value', [])]
-
-
+    
+    def update_list_item(self, item_id, update):
+        """Updates a list item with a given update
+           
+        :param item_id: List item id to update
+        :param update: Data to update list item with {'col_name': 'Updated Data'}
+        
+        :rtype: SharepointListItem
+        """
+        
+        url = self.build_url(self._endpoints.get('get_item_by_id').format(item_id=item_id) + '/fields')
+        
+        response = self.con.patch(url, update)
```

Comment:
You should check if the response succeeded.

like so:

```python
if not response:
    # handle the return default when there is a Http error: For example return False to flag this operation as a fail.
```

Reply:
changed to return false if the update fails and true if it works.